alcs-api/src/common/authorization/authorization.service.ts

```diff
@@ -110,31 +131,40 @@ export class AuthorizationService {
   }
 
   // FIXME: this will be replaced with automapper
```

Comment:
is it a good time to address this?

Reply:
I think that since the field names don't match, as in we have to change from snake_case to camelCase, I'll remove this comment.

Reply:
TODO removed.